File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/MuscleStrengthMeasuringFormViewModel.ts
//$C2E90999
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { MuscleStrengthMeasuringForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class MuscleStrengthMeasuringFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.MuscleStrengthMeasuringFormViewModel, Core';
    @Type(() => MuscleStrengthMeasuringForm)
    public _MuscleStrengthMeasuringForm: MuscleStrengthMeasuringForm = new MuscleStrengthMeasuringForm();
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/MuscleStrengthMeasuringFormViewModel.cs
//$04F0F22A
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class MuscleStrengthMeasuringFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class MuscleStrengthMeasuringFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var muscleStrengthMeasuringFormService = new MuscleStrengthMeasuringFormServiceController();
            muscleStrengthMeasuringFormService.MuscleStrengthMeasuringFormInternal(this, objectContext);
            return (MuscleStrengthMeasuringForm)objectContext.GetObject(this._MuscleStrengthMeasuringForm.ObjectID, "MuscleStrengthMeasuringForm");
        }
    }
}
